pyproject.toml

```
@@ -34,14 +34,12 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.9,<4"
-django-cms = ">=4.1.1"
+django-cms = "==5.0.0a1"
```

```suggestion
django-cms = ">=5.0.0a1"
```
Können wir hier nicht `>=` haben?

---

Done. Thanks for the hint.